src/main/java/com/spotinst/sdkjava/example/admin/organization/UserManagementUsageExample.java

```diff
@@ -280,4 +288,68 @@ private static Boolean deleteOrganization(SpotinstAdminOrganizationClient adminC
 
         return status;
     }
+
+    private static String createAccessPolicy(SpotinstAdminOrganizationClient adminClient) {
+        List<String> actions = new ArrayList<>();
+        actions.add("*");
+
+        List<String> resources = new ArrayList<>();
+        resources.add("*");
+
+        PolicyStatement.Builder statementBuilder = PolicyStatement.Builder.get();
+        PolicyStatement statements =
+                statementBuilder.setEffect(PolicyEffectEnum.ALLOW).setActions(actions).setResources(resources).build();
+
+        PolicyContent.Builder contentBuilder = PolicyContent.Builder.get();
+        PolicyContent         policyContent  = contentBuilder.setStatements(Collections.singletonList(statements)).build();
+
+        Policy.Builder policyBuilder = Policy.Builder.get();
+        Policy policy = policyBuilder.setName("Test Policy").setDescription("My test policy")
+                .setType(PolicyTypeEnum.ORGANIZATION).setPolicyContent(policyContent).build();
+
+        Policy createPolicyResponse = adminClient.createAccessPolicy(policy);
+
+        System.out.println(String.format("Created policy Id: %s", createPolicyResponse.getId()));
+        return createPolicyResponse.getId();
+    }
+
+    private static void updateAccessPolicy(String policyId, SpotinstAdminOrganizationClient adminClient) {
+        List<String> actions = new ArrayList<>();
+        actions.add("*");
```

Comment:
Can we take some more specific example in actions and policy? Just * will not be helpful to the customers referring these examples.